### PR TITLE
admin and reporting consolidation, and separation from ordering

### PR DESCRIPTION
**Pull request by caustin-usgs**

### CODE REVIEW

File: api/interfaces/admin/migrations/ordering_configuration_updates.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
+updict = dict()
```
Comment (klsmith-usgs) [inline api/interfaces/admin/migrations/ordering_configuration_updates.py]: is this used for anything?

File: api/interfaces/ordering/version0.py [hunk 2]
```diff
@@ -25,9 +23,6 @@ def __init__(self, providers=None):
         self.inventory = self.providers.inventory
         self.validation = self.providers.validation
         self.metrics = self.providers.metrics
-        self.production = self.providers.production
-        self.configuration = self.providers.configuration
-        self.reporting = self.providers.reporting
```
Comment (klsmith-usgs) [inline api/interfaces/ordering/version0.py]: I like this "trimming of the fat" that can now happen when things are split apart.

File: api/interfaces/admin/migrations/ordering_configuration_updates.py (hunk 1 above)
Comment (caustin-usgs) [inline api/interfaces/admin/migrations/ordering_configuration_updates.py]: not as far as i can tell

File: api/interfaces/admin/migrations/ordering_configuration_updates.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+for item in servd:
+    print base_sql.format(servd[item]['from'], servd[item]['to'], item)
+
+servd2 = {'url.tst.massloader': {'from':'MassLoader', 'to': 'MassLoaderdevmast'},
+ 'url.tst.orderdelivery': {'from':'OrderDeliveryService', 'to':'OrderDeliveryServicedevmast'},
+ 'url.tst.orderservice': {'from':'OrderWrapperService', 'to':'OrderWrapperServicedevmast'},
+ 'url.tst.orderupdate': {'from':'OrderStatusService', 'to':'OrderStatusServicedevmast'},
+ 'url.tst.registration': {'from':'RegistrationService', 'to':'RegistrationServicedevmast'}}
+
```
Comment (klsmith-usgs) [inline api/interfaces/admin/migrations/ordering_configuration_updates.py]: Still WiP I imagine?

Comment (caustin-usgs) [inline api/interfaces/admin/migrations/ordering_configuration_updates.py]: yea, i think this slipped in by accident

**Pull request merged**